azext_iot/central/services/device.py

```diff
@@ -744,7 +744,7 @@ def get_device_twin(
     """
 
     if not token:
-        aad_token = get_aad_token(cmd, resource="https://apps.azureiotcentral.com")[
+        aad_token = get_aad_token(cmd.cli_ctx, resource="https://apps.azureiotcentral.com")[
```

Comment:
Did this break due to the change in this PR https://github.com/Azure/azure-iot-cli-extension/pull/734/files#diff-498ed2ddec21bfd514f080fd3a7fd35d9af122fd2686fba4c0500d56a3ccfb11 ? What was the purpose of the change from `get_aad_token(cmd, ...) -> get_aad_token(cli_ctx, ...)` ?

Reply:
Oh yeah, probably.

I'm not sure of the reason for the change in the previous PR instead of just passing through `cmd`, but the pipeline was failing every central test with:

`AttributeError: 'AzCliCommand' object has no attribute 'cloud'` inside `Profile(cli_ctx=cli_ctx)`

Thinking we should revert it and pass `cmd` everywhere, and just access the `cli_ctx` property before it's used?